modules/ob-defining-annotations.adoc

```diff
@@ -0,0 +1,39 @@
+// This module is included in the following assembly:
+//
+// builds/configuring-build-strategies.adoc
+
+:_content-type: CONCEPT
```

Comment:
Almost all of the modules here seem more like reference modules than concept modules - maybe the content types need to be updated?

Reply:
I updated the content type to REFERENCE for all modules